README.md

```diff
@@ -46,9 +46,13 @@ jobs:
        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
 
 ```
+In case you are using Maven or Gradle scanner in your repository, you should alter the location of the report metadata file by using the optional `scanMetadataReportFile` input:
```

Comment:
```suggestion
In case you are using Maven or Gradle scanner in your repository, you should alter the location of the report metadata file by using the optional `scanMetadataReportFile` input.
```

As you also have a colon in the next sentence, you should remove this one.